app/src/main/java/com/duckduckgo/app/browser/WebNavigationState.kt

```diff
@@ -48,6 +49,9 @@ fun WebNavigationState.compare(previous: WebNavigationState?): WebNavigationStat
     if (this is EmptyNavigationState)
         return PageNavigationCleared
 
+    if (this is OverrideURL)
+        return Redirected
```

Comment:
This state is confusing, it sounds like a redirect happened but every normal load will go through this state.  I also worry that we haven't married the state change and the side-effects together e.g this causes the page to blank out but something different causes it to unblank. Let's talk about it on a call.

Reply:
Cool that you spotted this too. As we discussed on our call, I was not very happy either with this approach. I will remove this logic from here as I don't think is adding any value. thanks.